### PR TITLE
[T7][F12-B4]

### PR DESCRIPTION
**Pull request by bryanleejh**
Ready for review.

### CODE REVIEW

**Comment by bryanleejh** [comment]
Will continue updating the user guide over the next week as we morph the project towards the concept of bullet journal!

File: README.md [hunk 1]
```diff
@@ -1,9 +1,11 @@
...
 
-[![Build Status](https://travis-ci.org/se-edu/addressbook-level4.svg?branch=master)](https://travis-ci.org/se-edu/addressbook-level4)
-[![Build status](https://ci.appveyor.com/api/projects/status/3boko2x2vr5cc3w2?svg=true)](https://ci.appveyor.com/project/damithc/addressbook-level4)
-[![Coverage Status](https://coveralls.io/repos/github/se-edu/addressbook-level4/badge.svg?branch=master)](https://coveralls.io/github/se-edu/addressbook-level4?branch=master)
-[![Codacy Badge](https://api.codacy.com/project/badge/Grade/fc0b7775cf7f4fdeaf08776f3d8e364a)](https://www.codacy.com/app/damith/addressbook-level4?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=se-edu/addressbook-level4&amp;utm_campaign=Badge_Grade)
+[![Build Status](https://travis-ci.org/CS2103JAN2017-F12-B4/main.svg?branch=master)](https://travis-ci.org/CS2103JAN2017-F12-B4/main)
+[![Build status](https://ci.appveyor.com/api/projects/status/2fgsii7sa8ling2f?svg=true)](https://ci.appveyor.com/project/bryanleejh/main)
+[![Coverage Status](https://coveralls.io/repos/github/CS2103JAN2017-F12-B4/main/badge.svg?branch=master)](https://coveralls.io/github/CS2103JAN2017-F12-B4/main?branch=master)
```
Comment (okkhoy) [inline README.md:5]: :+1:

File: README.md [hunk 2]
```diff
@@ -1,9 +1,11 @@
...
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/129c2b2680614e0782cf9cc658797e83)](https://www.codacy.com/app/CS2103JAN2017-F12-B4/main?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=CS2103JAN2017-F12-B4/main&amp;utm_campaign=Badge_Grade)
+
+
```
Comment (okkhoy) [inline README.md:8]: the mock up/screenshot (below) should reflect your product, not address book any more

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -347,13 +347,47 @@ Priorities: High (must have) - `* * *`, Medium (nice to have)  - `* *`,  Low (un
...
-`* *` | user | hide [private contact details](#private-contact-detail) by default | minimize chance of someone else seeing them by accident
-`*` | user with many persons in the address book | sort persons by name | locate a person easily
-
-{More to be added}
+`* * *` | user | add events | help myself make tasks more manageable
+`* * *` | user | add tasks | help myself make tasks more manageable
+`* * *` | user | add deadlines | help myself make tasks more manageable
+`* * *` | user | CRUD events, tasks, deadlines | help myself make tasks more manageable
+`* * *` | user | undo last operation | help myself make tasks more manageable
+`* * *` | user | have some flexiblity in commands | help myself make tasks more manageable
```
Comment (okkhoy) [inline docs/DeveloperGuide.md:355]: can you be more specific on this?

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -347,13 +347,47 @@ Priorities: High (must have) - `* * *`, Medium (nice to have)  - `* *`,  Low (un
...
+`* * *` | user | simple search for finding an item | help myself make tasks more manageable
+`* * *` | user | view uncompleted/completed tasks | help myself make tasks more manageable
```
Comment (okkhoy) [inline docs/DeveloperGuide.md:357]: errr... most of your purpose fields are the same. what you have written is the over all goal. need to be specific for each user story

File: docs/UserGuide.md [hunk 5]
```diff
@@ -1,4 +1,4 @@
-# AddressBook Level 4 - User Guide
+# Main - User Guide
```
Comment (okkhoy) [inline docs/UserGuide.md:1]: Is the name of your product "Main"? If not, better to include the name here than `Main`.

Comment (okkhoy) [reply docs/UserGuide.md:1]: In the below, need to update the screenshot to a mock up of the UI you plan to implement. Later can be replaced by the actual screenshot.

File: docs/UserGuide.md [hunk 6]
```diff
@@ -47,102 +47,102 @@ Format: `help`
 
 > Help is also shown if you enter an incorrect command e.g. `abcd`
 
-### 2.2. Adding a person: `add`
+### 2.2. Adding a task: `add`
 
-Adds a person to the address book<br>
-Format: `add NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...`
+Adds a task to the address book<br>
```
Comment (okkhoy) [inline docs/UserGuide.md:52]: how about different variants? with deadlines, with start and end date/times etc?
need to capture all that the user can do

File: docs/UserGuide.md [hunk 7]
```diff
@@ -47,102 +47,102 @@ Format: `help`
...
-### 2.5. Finding all persons containing any keyword in their name: `find`
+### 2.5. Finding all tasks containing any keyword in their name: `find`
 
-Finds persons whose names contain any of the given keywords.<br>
+Finds tasks whose names contain any of the given keywords.<br>
 Format: `find KEYWORD [MORE_KEYWORDS]`
 
-> * The search is case sensitive. e.g `hans` will not match `Hans`
-> * The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
+> * The search is case sensitive. e.g `pick` will not match `Pick`
```
Comment (okkhoy) [inline docs/UserGuide.md:93]: think how this affects the user experience? 
in general, try to make the product as user friendly as possible, can?

File: docs/UserGuide.md [hunk 8]
```diff
@@ -152,27 +152,27 @@ Format: `exit`
...
 **Q**: How do I transfer my data to another Computer?<br>
 **A**: Install the app in the other computer and overwrite the empty data file it creates with
-       the file that contains the data of your previous Address Book folder.
+       the file that contains the data of your previous Task List folder.
 
 ## 4. Command Summary
 
-* **Add**  `add NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...` <br>
-  e.g. `add James Ho p/22224444 e/jamesho@gmail.com a/123, Clementi Rd, 1234665 t/friend t/colleague`
+* **Add**  `add TASK [t/TAG]...` <br>
```
Comment (okkhoy) [inline docs/UserGuide.md:166]: can consider providing the command summary as a table.

**Review by okkhoy — COMMENTED** [review]
This is a fair start to the user guide :-)

- it needs some more thinking through and polishing. i am sure, you guys will do the necessary changes before the final submission
- do take a look at the project scope and requirements, and ensure you have covered everything in that list here.
- please update the use cases to include a few related to your product.
- please ensure readme file reflects your project and not addressbook.
- please update the about us page to reflect your team.

File: docs/DeveloperGuide.md [hunk 9]
```diff
@@ -403,17 +437,66 @@ Use case ends.
 
 ## Appendix E : Product Survey
 
-**Product Name**
+**Google Calendar**
```
Comment (okkhoy) [inline docs/DeveloperGuide.md:440]: there is some formatting issues with your product surveys. please fix it.

**Comment by okkhoy** [comment]
@bryanleejh @arishuynhvan @zhypaul @WangYu-g 
Some comments added. Please ack & close the PR after reading comments.

For some reason, I missed reviewing it earlier. Apologies for that!
Good to see all of you have contributed to this exercise. Please keep up the good job.

File: docs/DeveloperGuide.md (hunk 4 above)
Comment (bryanleejh) [reply docs/DeveloperGuide.md:357]: @okkhoy This was due to a miscommunication! Will sort it out (: Thanks for highlighting this!

**Pull request closed without merging**